lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -535,6 +538,21 @@ def _delete_environment(self, environment: Environment, log: Logger) -> None:
             else:
                 log.debug("not wait deleting")
 
+    def _save_console_log(
+        self, resource_group_name: str, environment: Environment, log: Logger
+    ) -> None:
+        compute_client = get_compute_client(self)
+        vms = compute_client.virtual_machines.list(resource_group_name)
+        for vm in vms:
+            saved_path = environment.log_path / get_datetime_path() / vm.name
```

Comment:
1. don't call get_datetime_path multiple times in the loop, it will be different each time, it's not expected.
2. Don't make another level of folders of vm name. It doesn't need, because there are only one or two files for each vm.

Reply:
updated